tests/rbac/common/sawtooth/test_client_sync.py

```diff
@@ -42,42 +41,42 @@ def test_state(self):
         subtree = addresser.NAMESPACE
         for item in self.client.list_state(subtree=subtree)["data"]:
             address_type = item["address_type"] = addresser.address_is(item["address"])
-            if address_type == AddressSpace.USER:
+            if address_type == addresser.AddressSpace.USER:
```

Comment:
This sequence has a lot of if/else statements.  I would recommend a look up into key pair of a dict or something similar.  Fine as is, but something to consider

Reply:
Yeah, that test is skipped and temporary, there's a better way to deserialize data.